src/StatusQ/Components/StatusChatListAndCategories.qml

```diff
@@ -23,7 +23,6 @@ Item {
     property string selectedChatId: ""
```

Comment:
@alexandraB99 feel free to remove `selectedChatId` from both `StatusChatListAndCategories` and `StatusChatList` components.

Reply:
done

Reply:
Do we lose functionality in the sandbox app when this is removed? Cause I think it relies on this property, but would have to double check

Reply:
@PascalPrecht an item as well as subitem now have a property named `active`, so the old way `selected: model.chatId === statusChatList.selectedChatId` is now `selected: model.active`, and the value of `active` is maintained by the new refactored backend.